audioipc/src/lib.rs

```diff
@@ -140,26 +141,26 @@ pub(crate) unsafe fn duplicate_platform_handle(
         None
     };
 
-    let mut target_handle = std::ptr::null_mut();
-    let ok = handleapi::DuplicateHandle(
+    let mut target_handle = -1;
```

Comment:
Fixed, but note this needs to use the windows-sys version to be the correct type.

In windows-sys, the `HANDLE` type is represented as an `isize` rather than ` *mut c_void` as used in winapi and the Rust stdlib.  This means we're now forced to cast between these two representations somewhere, as done in these changes - I agree it's awkward in places, but I don't have a tidier solution right now.